### DISCUSSION TITLE
Binding name for Mouse wheel and double binding

### DISCUSSION THREAD

**Discussion by ExgalaxyFR** [r57zone/DualShock4-emulator#62] [Q&A]
Hello, as it's not present in the key name list, i would like to know if the mouse wheel up and down actions are bindable ?
I was also thinking of the possibility to use 2 bindings for the same gamepad button, the most common being the e and r for square as it mainains classic gaming standards for pc
Thank you for your work,
Exgalaxy

**Comment by r57zone** [comment]
Hi, at the moment, scrolling the wheel is supported, only pressing, I don’t know how to implement rotation yet, maybe I’ll add it in the future.
Pressing wheel - `MOUSE-MIDDLE-BTN`
